### ISSUE TITLE
Choice of uptime% on status page

### ISSUE THREAD

**Issue by aaruni96** [louislam/uptime-kuma#986]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

A choice about what uptime % (24 hour / 30 day / custom) is displayed for each monitor on the status page.

### ✔️ Solution

A global setting for status page, along with per monitor overrides for what uptime% is shown on the status page.

### 📝 Additional Context

I want to claim "99.95% uptime guaranteed", and I would like to show this to the public, without having to explain "I meant 99.95 uptime in the whole year, not in the past 24 hours as is shown on the status page"

**Comment by shinebayar-g** [comment]
This is a good one, right now status history is generated pretty much based only on interval, which results in different time range data for targets.

**Comment by amastelek** [comment]
Yes, would actually like both 24 hour and 30 day on the status page.

**Comment by BrammyS** [comment]
I'd to be able to show yearly uptime! :D

**Comment by MedPlexNi** [comment]
I love the idea and would love to get it added.

**Comment by luisxiaomai** [comment]
Hope for this date dimension to choose in status page which like https://demo.upptime.js.org
<img width="885" alt="Screen Shot 2022-03-10 at 11 13 53 AM" src="https://user-images.githubusercontent.com/6835229/157581772-47f7f1a6-0c57-4a52-a862-819e18c5291f.png">

**Comment by ouquoi** [comment]
Yes can very great to have this kind of view ! Thanks to the team ;)

**Comment by hanselsen** [comment]
This would be awesome. Is there a way to get this in the next release?

**Comment by jstrausd** [comment]
Is there any update to this feature request? It's 2024 and still no info about the state and no information in the milestones about this "important" feature?

**Comment by CommanderStorm** [comment]
We are also community oriented ^^
Our contribution guide [can be found here](https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md)
